src/client/src/orangehrmAdminPlugin/pages/languagePackage/languagePackage.vue

```diff
@@ -144,13 +144,14 @@ export default {
           slot: 'title',
```

Comment:
can you add `scoped` attribute as below, incase avoid style scope
`<style src="./language-package.scss" lang="scss" scoped></style>`

Reply:
### Important! Add this

Reply:
### Important! Add this